src/components/DynamicCard/examples.md

```diff
@@ -0,0 +1,25 @@
+## Examples
+
+### Basic usage with template
+
+```html
+<nosto-dynamic-card handle="awesome-product" template="product-card" variant-id="123456">
+Placeholder content while loading...
+</nosto-dynamic-card>
+```
+
+This example shows how to render a product using a Shopify template with a specific variant. The component fetches the product data and renders it using the specified template, with optional placeholder content shown during loading. This is useful for creating dynamic product cards that match your existing Shopify theme design.
```

Comment:
replace

The component fetches the product data and renders it using the specified template

with

The component fetches the product markup and rendered using the "product-card" Shopify Liquid template